app/Providers/AppServiceProvider.php

```diff
@@ -20,7 +20,9 @@ class AppServiceProvider extends ServiceProvider
     public function register()
     {
         if ($this->app->environment('local', 'testing', 'staging')) {
-          $this->app->register(DuskServiceProvider::class);
+            $this->app->register(\Laravel\Telescope\TelescopeServiceProvider::class);
```

Comment:
loading telescope breaks dusk and for whatever reason Dusk ignores the `TELESCOPE_ENABLED=false` env var. Remove it from 'testing'